python/Ganga/Lib/LCG/Grid.py

```diff
@@ -1,9 +1,9 @@
 import os
 import re
 import tempfile
-import time
+import datetime
 
-from Ganga.GPIDev.Credentials import getCredential
+from Ganga.GPIDev.Credentials2 import credential_store
```

Comment:
Wouldn't it be better to just replace the Credentials module with a new one rather than one with a new name? Why should external modules have to know about this?